app/views/users/show.html.erb

```diff
@@ -1 +1,16 @@
-<h1>This is for the show</h1>
\ No newline at end of file
+<h2>Details of a selected user</h2>
+
+<span><%= @user.photo %></span>
+<span><%= @user.name %></span>
+
+<p><%= @user.bio %></p>
+
+<% @user.return_recent_posts.each_with_index do |post, index|%>
+<p>Post #<%= index+1 %></p>
+<p>
+  <%= post.text%>
+</p>
+ <span>Likes : <%= post.likes_counter %></span>
+  <span>Comments : <%= post.comments_counter %></span>
+<% end %>
```

Comment:
- [ ] kindly make all your pages including this to look like the wireframe
your design :point_down: 
![Screenshot from 2023-06-14 23-00-17](https://github.com/Cabraham1/Blog/assets/66097371/3746b075-6c03-4441-ab48-f04baf251c08)
wireframe :point_down: 
![Screenshot from 2023-06-14 23-02-12](https://github.com/Cabraham1/Blog/assets/66097371/1c0dd180-2f0d-4140-bd1e-edd8baffe9f8)